latest/apidoc/manager/input/input.js

```diff
@@ -6,8 +6,8 @@ var com = {
 };
 com.qmino.miredot.restApiSource = {
```

Comment:
I see stuff has been reordered here by the generator which is hard to compare (typical issue!), but I presume you've had a good play around with it and everything looks good/same as . before?

Reply:
Yes, it is mostly the same, some new fields are added, but it is is working fine.